From 307b9f1b4c86a04a19f680bea238a8a7d1523d90 Mon Sep 17 00:00:00 2001
From: Michael Cho <michael@michaelcho.dev>
Date: Sat, 15 Aug 2026 00:23:00 -0400
Subject: [PATCH] support building with boost 1.92.0 (#9494)

Fixes build error seen with boost 1.92.0:
```
 error: no member named 'clear' in 'boost::python::detail::final_list_derived_policies<std::list<std::vector<int>>, true>'
  296 |             return DerivedPolicies::clear(container);
      |                    ~~~~~~~~~~~~~~~~~^
```

Due to boostorg/python@c574084154f908a2d14c4c24866982d527c15111
---
 Code/RDBoost/list_indexing_suite.hpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Code/RDBoost/list_indexing_suite.hpp b/Code/RDBoost/list_indexing_suite.hpp
index 4a3e368a1e1..5c2e5754d68 100644
--- a/Code/RDBoost/list_indexing_suite.hpp
+++ b/Code/RDBoost/list_indexing_suite.hpp
@@ -107,6 +107,8 @@ class list_indexing_suite
     container.erase(beg, end);
   }
 
+  static void clear(Container& container) { container.clear(); }
+
   static size_t size(Container& container) { return container.size(); }
 
   static bool contains(Container& container, key_type const& key) {
